src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/ManageApprenticesTests.cs

```diff
@@ -53,6 +54,140 @@ public class ManageApprenticesTests
 
             //Assert
             actualModel.SortedByHeaderClassName.Should().EndWith("das-table__sort--asc");
+        }
+
+        [Test, MoqAutoData]
+        public async Task Then_Will_Update_Search_Details_To_Cookies(
+           IndexRequest request,
+           IndexViewModel expectedViewModel,
+           [Frozen] Mock<IModelMapper> apprenticeshipMapper,
+           [Frozen] Mock<ICookieStorageService<IndexRequest>> cookieService,
+           ApprenticeController controller)
+        {
+            //Arrange
+            request.FromSearch = false;
+
+            apprenticeshipMapper
+                .Setup(mapper => mapper.Map<IndexViewModel>(It.IsAny<IndexRequest>()))
+                .ReturnsAsync(expectedViewModel);
+
+            //Act
+            await controller.Index(request);
+
+            //Assert
+            cookieService.Verify(x => x.Update(CookieNames.ManageApprentices, request, It.IsAny<int>()));
```

Comment:
Should we also verify that the Get isn't called as well?

Reply:
yep makes sense and no harm in checking it so its in.